### ISSUE TITLE
Valid digiKam RSS feed fails with "Unable to start HTTP connection"

### ISSUE THREAD

**Issue by marc-medley** [ViennaRSS/vienna-rss#1172]
The digiKam RSS feed fails to refressh with "Unable to start HTTP connection" in ViennaRSS.

```
Connecting to feed:https://digikam.org/index.xml
Error retrieving RSS feed: Unable to start HTTP connection
```

```
Last Refreshed: Never
```

The `feed:https://digikam.org/index.xml` passes the "Validate" check https://www.feedvalidator.org/check?url=feed:https://digikam.org/index.xml

Also, digiKam RSS feed loads OK in the Qt-based [QuietRSS](https://quiterss.org/).

digiKam RSS Feed: `feed:https://digikam.org/index.xml`
feed validation: https://www.feedvalidator.org/check?url=feed:https://digikam.org/index.xml

Vienna Version 3.4.2 :7bc20f57: (6905)
macOS High Sierra 10.13.6 (17G65)

**Comment by Eitot** [comment]
Might be resolved with #1165. This commit is not present yet in the release builds.

**Comment by marc-medley** [comment]
The workaround to manually remove `feed:` from `feed:https://...` does enable Vienna to load the RSS content sucessfully.

**Comment by barijaona** [comment]
The initial cause should be fixed by #1165
